catalog/MDCCatalog/AppTheme.swift

```diff
@@ -16,22 +16,25 @@ import Foundation
 import MaterialComponents.MaterialButtons_ButtonThemer
 import MaterialComponents.MaterialColorScheme
 import MaterialComponents.MaterialTypographyScheme
+import MaterialComponentsBeta.MaterialContainerScheme
 
 final class AppTheme {
-  let colorScheme: MDCColorScheming
```

Comment:
At the moment, these properties can be left in-place to reduce required code changes. We can do gradual migrations if we simply assign the `colorScheme` property of the `containerScheme` to be the same as the `colorScheme` property in this class. Same for TypographyScheme and ShapeScheme.